app/repositories/UserRespository.ts

```diff
@@ -0,0 +1,24 @@
+import User from "../models/user";
+import { AppResponse } from "../services/app-responses";
+import Logger from "../config/logger";
+import RepositoryError from "../errors/RepositoryError";
+
+export class UserRespository {
+  public response: AppResponse = new AppResponse();
+
+  public getAllUsers = async () => {
```

Comment:
@Suresh-ashwini This gives all the users list ..Should have to exclude the deleted users from the response

Reply:
get All Users is not returning the deleted users